source/testing/unit-testing-basics.md

```diff
@@ -33,25 +33,51 @@ should trigger the computed property), and assert that the logic in our
 computed property is working correctly.
 
 ```tests/unit/models/some-thing-test.js
-import { moduleFor, test } from 'ember-qunit';
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
 
-moduleFor('model:some-thing', 'Unit | some thing', {
-  unit: true
-});
+import { run } from '@ember/runloop';
+
+module('Unit | Model | some thing', function(hooks) {
+  setupTest(hooks);
 
-test('should correctly concat foo', function(assert) {
-  const someThing = this.subject();
-  someThing.set('foo', 'baz');
+  test('should correctly concat foo', function(assert) {
+    const someThing = run(() => this.owner.lookup('service:store').createRecord('some-thing'));
```

Comment:
The required `run` wrapping makes me sad.  Cross linking https://github.com/emberjs/data/issues/5247 with a discussion around whats going on. We have a path forward, just need a champion to tackle fixing it...